offline/web/mapEdditor.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
```

Comment:
I'm signing off on this file for now, one of the projects over the summer will be to revamp this into a more robust system